wrappers/golang/core/internal/helpers_test.go

```diff
@@ -4,27 +4,27 @@ import (
 	"math/rand"
 )
 
-func generateRandomLimb(size int) []uint32 {
-	limbs := make([]uint32, size)
+func generateRandomLimb(size int) []uint64 {
+	limbs := make([]uint64, size)
 	for i := range limbs {
-		limbs[i] = rand.Uint32()
+		limbs[i] = rand.Uint64()
 	}
 	return limbs
 }
 
-func generateLimbOne(size int) []uint32 {
-	limbs := make([]uint32, size)
+func generateLimbOne(size int) []uint64 {
+	limbs := make([]uint64, size)
 	limbs[0] = 1
 	return limbs
 }
 
-func generateBytesArray(size int) ([]byte, []uint32) {
-	baseBytes := []byte{1, 2, 3, 4}
+func generateBytesArray(size int) ([]byte, []uint64) {
+	baseBytes := []byte{1, 2, 3, 4, 5, 6, 7, 8}
 	var bytes []byte
-	var limbs []uint32
+	var limbs []uint64
 	for i := 0; i < size; i++ {
 		bytes = append(bytes, baseBytes...)
-		limbs = append(limbs, 67305985)
+		limbs = append(limbs, 578437695752307201)
```

Comment:
what is this number for?

Reply:
Its the decimal representation of baseBytes interpreted as a uint64